katka/signals.py

```diff
@@ -0,0 +1,33 @@
+from django.conf import settings
+from django.db.models.signals import post_save
+from django.dispatch import receiver
+
+from .constants import PIPELINE_STATUS_INITIALIZING, STEP_FINAL_STATUSES
+from .fields import username_on_model
+from .models import SCMPipelineRun, SCMStepRun
+
+
+@receiver(post_save, sender=SCMStepRun)
+def update_pipeline_steps_completed(sender, **kwargs):
```

Comment:
Should this not be step singular?

Reply:
No, but I see this is confusing, I'll change the name and add a docstring to clarify this